<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestorFinanciero</class>
 <widget class="QMainWindow" name="gestorFinanciero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor ACOES: Gestor de financiero</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lBusqueda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Busqueda:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tTransacciones">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>731</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cuantia</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Destino</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de emision</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Gasto</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>35</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>Honduras</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>2018-11-10</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bAtras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Atras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bAnadirTransaccion">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Anadir transaccion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bInforme">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>440</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Informe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bActualizar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lId">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTipo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCuantia">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cuantia</string>
    </property>
   </widget>
   <widget class="QLabel" name="lMoneda">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Moneda</string>
    </property>
   </widget>
   <widget class="QLabel" name="lDestino">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Destino</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFecha">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cSimboloCuantia">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>10</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cMoneda">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cDestino">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cSimboloFecha">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eId">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bConversor">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>440</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Conversor de Moneda</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cTipo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eCuantia">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFecha">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
